langchain4j-open-ai/src/main/java/dev/langchain4j/model/openai/InternalOpenAiHelper.java

```diff
@@ -35,14 +36,25 @@ public class InternalOpenAiHelper {
     static final String OPENAI_DEMO_URL = "http://langchain4j.dev/demo/openai/v1";
 
 
-    static final String DEFAULT_USER_AGENT = "langchain4j-openai";
```

Comment:
@jdubois same question here

Reply:
@langchain4j and same answer as above :-)